metadata-db-app/src/cmr/metadata_db/services/concept_service.clj

```diff
@@ -673,7 +687,7 @@
 
     (info "Starting deletion of old" concept-type-name "for provider" (:provider-id provider))
     (force-delete-with
-      context provider concept-type
+      context provider concept-type false
```

Comment:
The old revision that is deleted could be a tombstone. We can't treat it as not.